### DISCUSSION TITLE
Errors in Spanish Translation

### DISCUSSION THREAD

**Discussion by Cetykon** [CatimaLoyalty/Android#1854] [General]
Hello, I just came across this project, and this is something I will personally be using. But I noticed that some of the verb tenses in the Spanish translation needs work, along with a few incorrectly translated words.

(I will be helping with the translation on Weblate. When I am done, I will be submitting a pull request with the updated String.xml file, including comments with the previous translations and quality of life changes, so you can have an overall view of the changes.)

**Comment by TheLastProject** [comment]
Hi, thanks for helping! As I don't speak Spanish I sadly can't do much more than run translations through a translator to at least filter the most obvious bad translations, so yeah, this stuff can slip through :sweat_smile: 

Please fully use Weblate though if possible, no need to submit a pull request (as I can't review it beyond throwing it into a translator anyway) :)

For explaining your translations, you could probably use the "Comments" option in Weblate so other translators could understand why you made your changes. As I don't speak Spanish, even if you explain your changes to me, I can't really make any decisions based on that info. As I understand it, the only people who could really use that info would be possible other (future) Spanish translators, to keep a consistent style.

(I'll be converting this issue to a discussion because there's not really anything I could do about it, except review the upcoming MR, so it doesn't really need "tracking" in the traditional sense. Rest assured I'm not trying to end any discussion here :))

**Reply by Cetykon** [reply]
Ok, thanks for responding. I started to summit some chances and commenting on why I did them. Most changes were made to have language consistency throughout the app. But I am still working on it.
